src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLOR_COUNTER = Color.values().length;
```

Comment:
color values length is a constant by itself, so you can use without creating another one
```suggestion
```